include/mbedtls/config.h

```diff
@@ -256,6 +256,47 @@
  */
 //#define MBEDTLS_DEPRECATED_REMOVED
 
+/**
+ * \def MBEDTLS_CHECK_PARAMS
+ *
+ * This configuration controls whether the library validates more of the
```

Comment:
This configuration option / define sounds a bit better for me, but that's a minor.